pulsedrop_direct/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "codecId": "0032",
+  "sourceUrl": "https://github.com/the-real-vutility/payload-codecs/pulsedrop_direct",
+  "mac": "d46352",
+  "version": "1.0.0",
```

Comment:
Do you mind doing a minor version bump for the HDD codec in the metadata and package files? Something like 1.0.1 should be fine

Reply:
Sounds good... I updated the version attribute to 1.0.1 in the "metadata.json" and "package.json" files. Let me know if there's anywhere else I've missed.

Reply:
Looks good!

Reply:
Is there a way we can test this codec against a real PulseDrop? (like put into a temporary Grafana dashboard?)
I'd love to prove it out before merging if it isn't a big task.